src/mame/konami/konamigv.cpp

```diff
@@ -355,7 +377,22 @@ class tokimeki_state : public konamigv_state
 	uint16_t m_serial_sensor_data;
 	uint8_t m_heartbeat_signal;
 
+	uint8_t m_printer_bit;
+	uint8_t m_printer_data[6];
+	uint8_t m_printer_curbit;
+	uint8_t m_printer_curbyte;
+	attotime m_printer_pulse_starttime;
+	uint8_t m_printer_is_printing;
+	uint32_t m_printer_current_image;
+	bool m_printer_is_manual_layout;
+	bool m_printer_page_is_dirty;
+	int m_printer_video_last_vblank_state;
```

Comment:
`m_printer_video_last_vblank_state` is registered for save states, so it should be an explicitly sized integer or `bool`.